### PR TITLE
Add aria description and notify when loaded

### PR DESCRIPTION
**Pull request by max-hao-d2l**
* Add aria description to `outcomes-list`
* Notify if all outcomes in the user progress hierarchy list is rendered

### CODE REVIEW

File: src/outcomes-list/outcomes-list.js [hunk 1]
```diff
@@ -443,6 +461,21 @@ export class OutcomesList extends mixinBehaviors(
...
 	}
+
+	_onChildLoaded() {
+		this._loadedChildren++;
+		if (this._loadedChildren === this._outcomes.length) {
+			afterNextRender(this, () => {
+				const event = new CustomEvent('iron-announce', {
+					bubbles: true,
+					composed: true,
+					detail: { text: 'Loading Complete' }
```
Comment (rgoessl) [inline src/outcomes-list/outcomes-list.js]: Lang term

File: src/outcomes-list/outcomes-tree-node.js [hunk 2]
```diff
@@ -232,7 +232,8 @@ export class OutcomesTreeNode extends mixinBehaviors(
                 <d2l-outcomes-tree-node
                   href="[[_getSelfHref(item)]]"
                   token="[[token]]"
-                  has-parent=""
+				  has-parent=""
```
Comment (rgoessl) [inline src/outcomes-list/outcomes-tree-node.js]: indentation

File: src/outcomes-list/outcomes-tree-node.js [hunk 3]
```diff
@@ -364,6 +374,15 @@ export class OutcomesTreeNode extends mixinBehaviors(
 		this.addEventListener('blur', () => nodeData.blur());
 	}
 
+	_onOutcomeEntityChanged(outcomeEntity) {
+		if (outcomeEntity && !this._isLoaded) {
+			this._isLoaded = true;
```
Comment (rgoessl) [inline src/outcomes-list/outcomes-tree-node.js]: Couldn't this be triggered before all children are finished

Comment (max-hao-d2l) [reply src/outcomes-list/outcomes-tree-node.js]: True, I guess this should be in `_onChildLoaded` for parent nodes

File: src/outcomes-list/outcomes-list.js (hunk 1 above)
Comment (max-hao-d2l) [reply src/outcomes-list/outcomes-list.js]: Fixed

File: src/outcomes-list/outcomes-tree-node.js (hunk 2 above)
Comment (max-hao-d2l) [reply src/outcomes-list/outcomes-tree-node.js]: Fixed

**Comment by rgoessl** [comment]
Works good

**Pull request merged**